README.md

```diff
@@ -4,79 +4,73 @@ Gqrx
-including Funcube Dongle, RTL-SDR, Airspy, HackRF, BladeRF, RFSpace, USRP and
-SoapySDR.
-
-Gqrx can operate as an AM/FM/SSB receiver with audio output or as an FFT-only
-instrument. There are also various hooks for interacting with external
+Gqrx is an open source software defined radio (SDR) receiver implemented using [GNU Radio](https://gnuradio.org) and the [Qt GUI toolkit](https://www.qt.io/).
+Currently it works on Linux and Mac with hardware supported by gr-osmosdr, including Funcube Dongle, RTL-SDR, Airspy, HackRF, BladeRF, RFSpace, USRP and SoapySDR.
+
+Gqrx can operate as an AM/FM/SSB receiver with audio output or as an FFT-only instrument. There are also various hooks for interacting with external
 applications using network sockets.
 
 ![Screenshot of the main Gqrx window](resources/screenshots/gqrx-main.png)
 
 
 Download
---------
+========
 
-Gqrx is distributed as a source code package and binaries for Linux and Mac.
-Many Linux distributions provide gqrx in their package repositories.
-Alternate Mac support is available through [MacPorts](https://ports.macports.org/port/gqrx/summary) and [Homebrew](https://formulae.brew.sh/cask/gqrx).
+Gqrx is distributed as a source code package and binaries for Linux and Mac. Many Linux distributions provide gqrx in their package repositories.
 
 * [Official releases](https://github.com/gqrx-sdr/gqrx/releases)
 * [Pre-release builds](https://github.com/gqrx-sdr/gqrx/actions?query=workflow%3ABuild+branch%3Amaster)
 
-Usage
------
+Alternate Mac support is available through [MacPorts](https://ports.macports.org/port/gqrx/summary) and [Homebrew](https://formulae.brew.sh/cask/gqrx).
 
-It is strongly recommended to run the `volk_profile` utility before
-running gqrx. This will detect and enable processor-specific optimisations and
-will in many cases give a significant performance boost.
+Installation 
+============
 
-The first time you start gqrx it will open a device configuration dialog.
-Supported devices that are connected to the computer are discovered
-automatically and you can select any of them in the drop-down list.
+Our installation instructions cover Linux, Mac OS, Raspberry-Pi, and source code. If you're installation option is not available here, you can request it via [this 1 question poll](https://poll.fm/11102781).
 
-If you don't see your device listed in the drop-down list it could be because:
-- The driver has not been included in a binary distribution
-- The udev rule has not been properly configured
-- Linux kernel driver is blocking access to the device
+Linux distributions
+-------------------
 
-You can test your device using device specific tools, such as rtl_test,
-airspy_rx, hackrf_transfer, qthid, etc.
+The simplest installation for Linux distributions is by our [official AppImage release](https://github.com/gqrx-sdr/gqrx/releases). Many Linux distributions provide gqrx in their package repositories. Linux distribution versions can be out-of-date. 
```

Comment:
Here it might be worth to hint to give feedback to maintainers to push for newer releases..

Reply:
Yup. I think best approach to this would be to create a bug report issue template? Thoughts? Before I go create one?